src/MOIWrapper.jl

```diff
@@ -82,6 +82,19 @@ function replace_inf(x::Real)
     end
 end
 
+
+function LQOI.get_objectivesense(model::Optimizer)
+    s = get_obj_sense(model.inner)
+    if s == 1.0
+        return MOI.MIN_SENSE
+    elseif s == -1.0
+        return MOI.MAX_SENSE
+    else
+        error("Invalid objective sense: $(s)")
```

Comment:
Is it possible for `get_obj_sense` to return something else? Can we add a test?

Reply:
Couldn't it be `MOI.FEASIBILITY_SENSE` ?

Reply:
According to https://projects.coin-or.org/Clp/browser/trunk/Clp/src/Clp_C_Interface.h#L419, it looks like a return value of 0.0 maps to `MOI.FEASIBILITY_SENSE` (ignore the objective).

Reply:
Thank you for catching that and bonus for pointing me to the location in the Clp source!  Just pushed a fix.

@odow I briefly looked add the idea of adding a test, but seems that the tests are owned by MOI -- would that be the place to add such a test?

Reply:
Just add a new test to the bottom of https://github.com/JuliaOpt/Clp.jl/blob/master/test/MOIWrapper.jl.
Create a model, set the objective sense to -3, and then query `LQOI.get_objective_sense`. Alternatively, just change the `elseif` prior to the `FEASIBILITY_SENSE` into an `else` and remove the error.

Reply:
Thank @odow -- I just pushed a change with your alternative suggestion.